### DISCUSSION TITLE
The problem with removing a new migration

### DISCUSSION THREAD

**Discussion by khizbatov** [jasontaylordev/CleanArchitecture#970] [Q&A] [answered]
**Describe the bug**
When trying to remove a new migration, an error occurs:
`The migration '20230809110208_Test' has already been applied to the database. Revert it and try again. If the migration has been applied to other databases, consider reverting its changes using a new migration instead.`

**To Reproduce**

1. Create a solution
2. Set `false` for the `UseInMemoryDatabase` property in the `appsettings.json`
3. To add any new field for the class `ToDoItem.cs` as an example
4. `Add-Migration Test -OutputDir Persistence\Migrations` in Package Manager Console
5. `Update-Database`
6. `Update-Database 0`
7. `Remove-Migration` ... error

It seems that the problem is caused by NSwag.

**Comment by MedBouc** [comment]
What about
Remove-Migration -Force

**Reply by jasontaylordev** [reply, marked as answer]
Also, try building separately first. NSwag runs on build of WebUI, which occurs when trying to remove a migration. This might give you enough info to resolve the problem.

If that doesn't work, try commenting out the NSwag task in WebUI.csproj. Then run the build again and try to identity any issues. 

Let me know how you go.
